plat/nvidia/tegra/common/tegra_pm.c

```diff
@@ -146,18 +146,36 @@ void tegra_get_sys_suspend_power_state(psci_power_state_t *req_state)
  ******************************************************************************/
 void tegra_cpu_standby(plat_local_state_t cpu_state)
 {
+	u_register_t saved_scr_el3;
+
 	(void)cpu_state;
 
 	/* Tegra SoC specific handler */
 	if (tegra_soc_cpu_standby(cpu_state) != PSCI_E_SUCCESS)
 		ERROR("%s failed\n", __func__);
 
+	saved_scr_el3 = read_scr_el3();
+
+	/*
+	 * As per ARM ARM D1.17.2, any physical IRQ interrupt received by the PE
+	 * will be treated as a wake-up event, if SCR_EL3.IRQ is set to '1',
+	 * irrespective of the value of the PSTATE.I bit value.
+	 */
+	write_scr_el3(saved_scr_el3 | SCR_IRQ_BIT);
+
 	/*
 	 * Enter standby state
-	 * dsb is good practice before using wfi to enter low power states
+	 * isb & dsb is good practice before using wfi to enter low power states
 	 */
+	isb();
```

Comment:
Normally you would add ``isb()`` after ``dsb()`` so that the effect of the ``dsb()`` is seen.